shakedown/cli/main.py

```diff
@@ -11,6 +13,10 @@
 @click.command('shakedown')
 @click.argument('tests', nargs=-1)
 @click.option('-u', '--dcos-url', help='URL to a running DC/OS cluster.')
+@click.option('-U', '--provisioner-url', help='URL to a DC/OS provisioning service')
+@click.option('-P', '--provisioner-platform', help='Platform type to request from provisioning service')
+@click.option('--provisioner-wait', default=60, help='Seconds (exponential) to wait between provisioning attempts')
```

Comment:
Perhaps the available options flags should be left to the provisioner class.  For vmpooler it's a pre-defined set that can be requested from; for dcos-launch it'd make sense to send along a YAML/JSON payload to specify cluster type/shape.